tests/src/utils/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { KeyringPair } from '@polkadot/keyring/types';
+import { SubmittableExtrinsic } from '@polkadot/api/types';
+import BN = require('bn.js');
+
+export class Utils {
+  public static async signAndSend(
+    tx: SubmittableExtrinsic<'promise'>,
+    account: KeyringPair,
+    nonce: BN,
```

Comment:
I think having the nonce argument here is complicating the calls. Why not drop it and let this method manage the nonce. This way when it comes to adding ability to send multiple transactions within the same block the functionality can be added here.

Reply:
I made a lousy attempt in our storage node here https://github.com/Joystream/storage-node-joystream/blob/master/packages/runtime-api/index.js#L162 It needs a lot of improvement, but I've been very reluctant to touch that code because of lack of tests. Hopefully you could make a better implementation here in the tests and we can reuse it in the storage node.

Reply:
Please see `utils/sender.ts`